application/views/admin/export/vv_view.php

```diff
@@ -88,8 +78,30 @@
                                     <?php eT("Old VV version"); ?>
                                 </label>
                             </div>
+                            <p class="help-block"><?php eT("If you want to import survey on old installation or if your survey have problem: use old version (automatically selected if some code are duplicated)."); ?></p>
```

Comment:
Please correct to:
"If you want to import the response data in older version or if your survey has integrity problem, please use the old export version (automatically selected if there are duplicate codes)."